sql/hive/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -1,140 +0,0 @@
-/*
```

Comment:
yes. These are the two files which were copied over, then allowed to age while the originals were maintained.

1. cut these files and things don't build any more
1.  add the mvn changes and they do compile, except where `CachedTableSuite` had another method from the original tests pasted in.
1. remove that method, the updated parent class exports the method, and all is well.